plugins/scaffolder-backend/src/scaffolder/stages/publish/github.ts

```diff
@@ -79,6 +80,25 @@ export class GithubPublisher implements PublisherBase {
 
     const { data } = await repoCreationPromise;
 
+    if (access && access.match(new RegExp(`${owner}/.+`))) {
```

Comment:
Would ``access?.startsWith(`${owner}/`)`` work here? or `.includes()`

Reply:
Done, raised https://github.com/spotify/backstage/pull/2492 as when I went to retest this master wasn't working.